source/scss/_pf-basic-variables.scss

```diff
@@ -456,3 +460,33 @@ $tag-font-weight:           $pf-font-weight-bold;
 // Use a higher than normal value to ensure completely rounded edges when
 // customizing padding or font-size on labels.
 // $tag-pill-border-radius:    10rem !default;
+
+
+// Form states and alerts
+//
+// Define colors for form feedback states and, by default, alerts.
+
+$state-success-text:             $color-pf-black-800;
+$state-success-bg:               lighten($color-pf-green-100, 8%);
+$state-success-border:           $brand-success;
+
+$state-info-text:                $color-pf-black-800;
+$state-info-bg:                  $color-pf-black-150;
+$state-info-border:              $color-pf-black-500;
+
+$state-warning-text:             $color-pf-black-800;
+$state-warning-bg:               lighten($color-pf-orange-100, 8%);
+$state-warning-border:           $brand-warning;
+
+$state-danger-text:              $color-pf-black-800;
+$state-danger-bg:                lighten($color-pf-red-100, 55%);
+$state-danger-border:            $brand-danger;
+
+
+// Alerts
+//
+// Define alert colors, border radius, and padding.
+
+$alert-padding-x:             $spacer;
+$alert-link-font-weight:      normal;
+$alert-close-font-size:       1.5rem;
```

Comment:
Actually I was using this variable for the icon size in alerts. So I updated the variable name to $pf-alert-icon-font-size:     $icon-size-sm; 
Introduced the icon-size-sm variable, since we have other icon-size- and none of the font-sizes fit.